spec/OpenLineage.json

```diff
@@ -443,6 +446,41 @@
         }
       ]
     },
+    "ExpectationsAssertionsDatasetFacet": {
```

Comment:
Maybe minor consideration- expectation and assertion seem redundant to me. The `ExpectationsAssertion` wording seems to reference the Great Expectations framework specifically, whereas I think something like just `DataQualityAssertionsDatasetFacet` is more general. Similarly, below, I'd change `expectationType` to `assertion`- reserve `expectation` for fields that contrast `expected` and `actual`, as we'd likely encounter them in any testing framework.

Reply:
agreed, the name `DataQualityAssertions` is more intuitive. The field above would become `dataQualityAssertions`

Reply:
Done